src/dsp_tools/models/custom_warnings.py

```diff
@@ -1,7 +1,20 @@
 from typing import Protocol
 from typing import runtime_checkable
 
-from termcolor import cprint
+# These are ANSI escape codes which can be used to configure the print output on the terminal
+# https://en.wikipedia.org/wiki/ANSI_escape_code
+# The semicolon separates different configurations
+
+# All codes must start with "\033["
```

Comment:
I have taken it apart, so that for the future it will be easier to understand the construct.

Reply:
Very good introduction to ANSI codes, thanks :-)